code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Other.dm

```diff
@@ -354,6 +351,20 @@
 			return
 	M.clean_blood()
 
+
+	if(istype(M,/mob/living/carbon/slime))
+		var/mob/living/carbon/slime/S = M
+		//Adult slimes have 200 HP, baby slimes have 100 HP.
+		//20u of cleaner should kill a baby slime in 10 seconds.
+		//Wiki: Deals volume damage to slimes per second.
+		S.adjustToxLoss( (volume/20) * (removed/REM) * 100 * (1/10) )
+		if(!S.client)
```

Comment:
I am not sure if those changes wouldnt make it a bit too easy to deal with slime outbreaks.

Reply:
Reduced